Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -4920,6 +4920,9 @@ public void CustomizationsContainStableFiniteSequenceRelayByDefault()
         [InlineData(typeof(ICollection<>), typeof(List<>))]
         [InlineData(typeof(IReadOnlyCollection<>), typeof(ReadOnlyCollection<>))]
         [InlineData(typeof(IDictionary<,>), typeof(Dictionary<,>))]
+#if NET5_0_OR_GREATER
+        [InlineData(typeof(IReadOnlySet<>), typeof(HashSet<>))]
```

Comment:
It's odd that there are no tests that assert the types are actually relayed to the correct type. Only that the relay exists in the fixture.

Reply:
It shouldn't be difficult to add something that uses the Fixture to create an interface and checks the concrete type.

Reply:
yep it's basically the same setup but actually invoking the fixture.
```cs
    [Theory]
    [InlineData(typeof(IList<>), typeof(List<>))]
    [InlineData(typeof(IReadOnlyList<>), typeof(ReadOnlyCollection<>))]
    [InlineData(typeof(ICollection<>), typeof(List<>))]
    [InlineData(typeof(IReadOnlyCollection<>), typeof(ReadOnlyCollection<>))]
    [InlineData(typeof(IDictionary<,>), typeof(Dictionary<,>))]
#if NET5_0_OR_GREATER
    [InlineData(typeof(IReadOnlySet<>), typeof(HashSet<>))]
#endif
    public void ResidueCollectorsContainForwardsByDefault(Type request, Type expected)
    {
        // Arrange
        var sut = new Fixture();
        var context = new SpecimenContext(sut);

        // Act
        var actual = context.Resolve(request);

        // Assert
        Assert.IsType(expected, actual);
    }
```
Could you add the test in the PR?

Reply:
You took the fun away :P